backend/server.js

```diff
@@ -45,10 +44,13 @@ sbp('sbp/selectors/register', {
   'backend/server/handleEntry': async function (entry: GIMessage) {
     const contractID = entry.contractID()
     await sbp('gi.db/log/addEntry', entry)
-    const response = makeResponse(RESPONSE_TYPE.ENTRY, entry.serialize())
     console.log(chalk.blue.bold(`broadcasting ${entry.description()}`))
-    sbp('okTurtles.data/apply', PUBSUB_INSTANCE, p => {
-      p.room(contractID).write(response)
+    sbp('okTurtles.data/apply', PUBSUB_INSTANCE, pubsub => {
+      if (contractID in pubsub.subscribersByContractID) {
```

Comment:
I agree that an accompanying type annotation could have made it clearer, but yes this is the intended behavior.

Every property value found on the `pubsub.subscribersByContractID` object must be a `Set` of contract IDs.

Otherwise, if unexpected properties or property values can get here, an error should be reported, preferably at build time.

However, the check is still needed because a matching property for the given contract ID might not have been defined here yet, for example if there are no subscriber yet because the contract just got created and the `GIMessage` being handled is the very first one of the contract.